scipy/special/cephes/yv.c

```diff
@@ -1,47 +1,12 @@
-/*                                                     struve.c
- *
- *      Struve function
- *
- *
- *
- * SYNOPSIS:
- *
- * double v, x, y, struve();
- *
- * y = struve( v, x );
- *
- *
- *
- * DESCRIPTION:
- *
- * Computes the Struve function Hv(x) of order v, argument x.
- * Negative x is rejected unless v is an integer.
- *
- * This module also contains the hypergeometric functions 1F2
- * and 3F0 and a routine for the Bessel function Yv(x) with
- * noninteger v.
- *
- *
- *
- * ACCURACY:
- *
- * Not accurately characterized, but spot checked against tables.
- *
- */
-
-
-/*
- * Cephes Math Library Release 2.81:  June, 2000
- * Copyright 1984, 1987, 1989, 2000 by Stephen L. Moshier
- */
```

Comment:
The copyright shouldn't be removed I think, as it probably applies to the routines.

Reply:
Right. Put it back in.